src/HotChocolate/Language/src/Language.Visitors/Contracts/ISyntaxVisitor.cs

```diff
@@ -8,4 +6,12 @@ public interface ISyntaxVisitor
             ISyntaxNode node,
             ISyntaxVisitorContext context);
     }
+
+    public interface ISyntaxVisitor<TContext>
```

Comment:
now i have to rewrite my visitors again :D

Reply:
why ... you can still use ISyntaxVisitor we now just have a generic version of it as well.